playbook/app/pb_kits/playbook/pb_phone_number_input/_phone_number_input.scss

```diff
@@ -0,0 +1,90 @@
+@import "../tokens/colors";
+
+.pb_phone_number_input {
+  input::placeholder {
+    color: $focus_input_light;
+  }
+
+  .iti__country.iti__highlight {
+    background-color: $hover_light;
+
+    .iti__country-name, .iti__dial-code {
+      color: $primary
+    }
+  }
+
+  .iti--allow-dropdown .iti__flag-container:hover .iti__selected-flag {
+    background-color: $focus_input_light;
+  }
+
+  .iti__flag-container:hover + .text_input {
+    background-color: $focus_input_light;
+  }
+
+  .iti__flag {
+    background-image: url("https://cdnjs.cloudflare.com/ajax/libs/intl-tel-input/17.0.19/img/flags.png");
+  }
+
+  .iti--separate-dial-code .iti__selected-flag {
+    background-color: rgba(0, 0, 0, 0);
+  }
+
+  .iti__arrow {
+    border-left: unset;
+    border-right: unset;
+    border-top: unset;
+    width: unset;
+    height: unset;
+    margin-bottom: 4px;
+  }
+
+  .iti__arrow--up {
+    border-bottom: unset;
+  }
+
+  .iti__arrow::before {
+    border-style: solid;
+    border-width: 0.1em 0.1em 0 0;
+    content: '';
+    display: inline-block;
+    height: 0.6em;
+    left: 0.1em;
+    position: relative;
+    vertical-align: top;
+    width: 0.6em;
+    top: 10px;
+    transform: rotate(135deg);
+    font-size: 0.5em;
+    color: $slate;
+  }
+
+  .iti__arrow.iti__arrow--up::before {
+    transform: rotate(-45deg);
+    top: 12px;
+  }
+
+  .iti__active::after {
+    float: right;
+    content: "";
+    margin-top: 5px;
+    transform: rotate(45deg);
+    height: 12px;
+    width: 6px;
+    border-bottom: 2px solid;
+    border-right: 2px solid;
+    border-radius: 1px;
+  }
+
+  .iti__country-list {
+    min-width: 340px;
```

Comment:
Could be on opportunity to add [variables to these](https://github.com/powerhome/playbook/blob/master/playbook/app/pb_kits/playbook/utilities/_max_width.scss) and use them here

Reply:
@jasoncypret, in this case, this size is very specific to the phone number input. It determines the smallest size the dropdown needs to have so that the country with the longest name doesn't break the layout. I don't believe it will be reused globally. In this case, would it be better to create a variable inside the **__phone_number_input.scss_** file?